### PR TITLE
Metric Exporters Prep - Rename ConfigurationKeys Variables

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Renames variables in `ConfigurationKeys.cs`

### CODE REVIEW

File: src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs [hunk 1]
```diff
@@ -50,27 +50,27 @@ public class ConfigurationKeys
     /// <summary>
     /// Configuration key for whether the trace console exporter is enabled.
     /// </summary>
-    public const string ConsoleExporterEnabled = "OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED";
```
Comment (zacharycmontoya) [inline src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs:53]: What do you think of creating a subclass `ConfigurationKeys.Tracer` and moving most/all fields into that subclass?

Comment (rajkumar-rangaraj) [reply src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs:53]: Fixed bdfbbdec76779309882e3b79e17cc8fa877facea

**Review by nrcventura — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Review by RassK — APPROVED** [review]

File: src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs [hunk 2]
```diff
@@ -97,6 +49,60 @@ public class ConfigurationKeys
...
+        /// <summary>
+        /// Configuration key for the traces exporter to be used.
+        /// Default is <c>"otlp"</c>.
+        /// </summary>
+        public const string TracesExporter = "OTEL_TRACES_EXPORTER";
+
+        /// <summary>
+        /// Configuration key for whether the trace console exporter is enabled.
+        /// </summary>
+        public const string ConsoleExporterEnabled = "OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED";
```
Comment (pellared) [inline src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs:78]: I think we should rename `OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED` to `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`.

The same comment applies to other places - we should rename rename `_TRACE_` to `_TRACES_`

Reference: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md

Comment (rajkumar-rangaraj) [reply src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs:78]: Should we use `OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP` or `OTEL_DOTNET_AUTO_LOAD_TRACES_AT_STARTUP`?

Comment (rajkumar-rangaraj) [reply src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs]: Fixed dc8ce26a0e501cebef0b66399b0826ec094755ee

**Review by nrcventura — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Pull request merged**